website/layout/DocsLayout.js

```diff
@@ -36,8 +36,7 @@ var DocsLayout = React.createClass({
     return (
       <Site
         section="docs"
-        title={metadata.title}
-        description={content.trim().split('\n')[0]} >
```

Comment:
The issue in the earlier PR was introduced here, as content can be either undefined or JSX. On further review, using the default website description when a doc is shared should be alright, so I'm removing this.